src/main/java/racing/domain/car/RacingCars.java

```diff
@@ -1,42 +1,43 @@
 package racing.domain.car;
 
-import java.util.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
 
 public class RacingCars {
     private static final String DELIMITER = ",";
-
-    private final List<Car> carPositions;
+    private final List<Car> cars;
 
     public RacingCars(String inputName) {
-        this.carPositions = prepareCars(inputName);
+        this.cars = prepareCars(inputName);
     }
 
     public List<Car> prepareCars(String inputName) {
-        List<Car> carsPositions = new ArrayList<>();
+        List<Car> cars = new ArrayList<>();
 
         checkInputName(inputName);
 
         String[] carNames = inputName.split(DELIMITER);
 
         for (int i = 0; i < carNames.length; i++) {
-            carsPositions.add(new Car(carNames[i]));
+            cars.add(new Car(carNames[i]));
         }
-        return carsPositions;
+        return cars;
     }
 
     private void checkInputName(String inputName) {
-        if (Objects.isNull(inputName) || inputName.equals(" ")) {
+        if (Objects.isNull(inputName) || inputName.trim().isEmpty()) {
             throw new IllegalArgumentException("이름이 비어있습니다. 이름을 입력해주세요.");
         }
     }
 
     public void moveCars(CarMoveBehavior carMoveBehavior) {
-        for (int i = 0; i < carPositions.size(); i++) {
-            carPositions.get(i).move(carMoveBehavior);
+        for (int i = 0; i < cars.size(); i++) {
+            cars.get(i).move(carMoveBehavior);
         }
```

Comment:
```suggestion
        List<Car> executedCars = new ArrayList<>();
        for (Car car : cars) {
            executedCars.add(car.move(carMoveBehavior));
        }
        return new RacingCars(executedCars);
```
자동차들이 움직인 상태를 새로운 객체로 선언하여 주소 참조를 끊어주세요 !